devtools/x/src/installer.rs

```diff
@@ -51,3 +94,19 @@ pub fn check_installed(name: &str, version: &str) -> bool {
     );
     found
 }
+
+fn install_all(tools: &[(String, String)]) -> bool {
+    let mut success: bool = true;
+    for (key, value) in tools {
+        success = success && install_if_needed(key, value);
```

Comment:
This should also work.

```suggestion
        success &= install_if_needed(key, value);
```